src/elements.ts

```diff
@@ -159,8 +159,6 @@ export class PermitElements {
       } else {
         iframeUrl = `${loginUrl}?tenant=${tenant}`
       }
-    } else {
```

Comment:
This is important for a case that the tenant is undefined, opened another PR here to fix it
https://github.com/permitio/permit-js/pull/21